tests/Integration/Infrastructure/Controller/Security/LoginControllerTest.php

```diff
@@ -43,7 +43,7 @@ public function testLoginAsAdminSuccessfully(): void
         $client->submit($form);
         $this->assertResponseStatusCodeSame(302);
         $crawler = $client->followRedirect();
-        $this->assertSame('Arrêtés de circulation Votre avis Administration Se déconnecter', $crawler->filter('[data-testid="user-links"]')->text());
+        $this->assertSame('Arrêtés de circulation Votre avis Mon espace', $crawler->filter('[data-testid="user-links"]')->text());
```

Comment:
ETQ super-admin le bouton Administration devrait toujours être présent (il disparaît dans l'état actuel de la PR et n'est pas remis nulle part sauf erreur de ma part)

```suggestion
        $this->assertSame('Arrêtés de circulation Votre avis Mon espace Administration', $crawler->filter('[data-testid="user-links"]')->text());
```

Reply:
Même remarque qu'ici https://github.com/MTES-MCT/dialog/pull/959#discussion_r1764677551